.buildkite/pipeline.yml

```diff
@@ -10,7 +10,7 @@ steps:
       platform: x86_64.metal
     plugins:
       - docker#v3.0.1:
-          image: "rustvmm/dev:v5"
+          image: "rustvmm/dev:v6"
```

Comment:
Does it make sense/is it possible to have a variable that defines the desired version of the container so we don't have to explicitly mention it in multiple places? (maybe we can even share it among multiple files)? I think I saw something like this somewhere, but I might be wrong :-s

Reply:
You saw it in a comment and a bunch of readmes. The closest thing I found to applying this to the pipeline is regenerating the pipeline at every run: https://buildkite.com/docs/pipelines/defining-steps#dynamic-pipelines
I can't figure out another way of substituting variables in the yaml.

Reply:
Sure, maybe there's some buildkite-specific or other feature, but we can look out for that in a separate setting. Looks good, it seems we just have fix some things the CI's complaining about.